### ISSUE TITLE
Could not resolve "mux-embed" error when using Image component

### ISSUE THREAD

**Issue by RhyG** [datocms/datocms-svelte#8]
After following the [installation instructions](https://github.com/datocms/datocms-svelte?tab=readme-ov-file#installation) and trying to use the image component 

```
<script>
	import { Image } from '@datocms/svelte';
</script>
```

I am getting this error:
```
Error: Build failed with 1 error:
node_modules/@datocms/svelte/package/components/VideoPlayer/VideoPlayer.svelte:28:16: ERROR: Could not resolve "mux-embed"
    at failureErrorWithLog (/Users/username/Documents/projects/datocms-svelte-repro/node_modules/esbuild/lib/main.js:1651:15)
    at /Users/username/Documents/projects/datocms-svelte-repro/node_modules/esbuild/lib/main.js:1059:25
    at /Users/username/Documents/projects/datocms-svelte-repro/node_modules/esbuild/lib/main.js:1527:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
Which looks to be because I don't have a dependency of the video player installed, even though I'm not using that component.

I've created a barebones repro [here](https://github.com/RhyG/datocms-svelte-repro), but issue can be easily reproduced by installing `@datacms/svelte` and importing the image component.

Thanks!

**Comment by arcataroger** [comment]
Sorry about that, @RhyG, and thanks for the detailed notes and example repo! I'll let the devs know and we'll provide an update as soon as we can.

**Comment by sistrall** [comment]
@RhyG I just released version [2.0.2](https://github.com/datocms/datocms-svelte/releases/tag/v2.0.2) that fixes the problem.

Thanks for reporting the issue and for the clear reproduction.

**Comment by RhyG** [comment]
Wow that was quick, thanks heaps!
